attributes/default.rb

```diff
@@ -1,5 +1,5 @@
 # osquery version and packs.
-default['osquery']['version'] = '2.3.2'
+default['osquery']['version'] = '2.4.0'
```

Comment:
Since the naming of osquery package is inconsistent, why not add OS version in version attribute? 
```
default['osquery']['version'] = '2.4.0-1.linux'
```
Unless we have any other place will refer to version number only `2.4.0`.